lib/universal-search.js

```diff
@@ -88,14 +88,27 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
+    console.log(osString);
 
     const stylesheet = doc.createElementNS('http://www.w3.org/1999/xhtml', 'h:link');
     stylesheet.rel = 'stylesheet';
-    stylesheet.href = 'chrome://universalsearch-skin/content/style.css';
     stylesheet.type = 'text/css';
     stylesheet.id = 'universal-search-stylesheet';
     stylesheet.style.display = 'none';
 
+    // If OS is Mac, load style-osx.css.
+    if (osString == 'Darwin') {
+      stylesheet.href = 'chrome://universalsearch-skin/content/style-osx.css';
+    //If OS is Linux, load style-linux.css.
+    } else if (osString == 'Linux') {
+      stylesheet.href = 'chrome://universalsearch-skin/content/style-linux.css';
+    //Otherwise, load style-windows.css.
+    } else {
+      stylesheet.href = 'chrome://universalsearch-skin/content/style-windows.css';
+    }
+
     docEl.appendChild(stylesheet);
```

Comment:
@kaganjd This looks great, aside from manually loading two stylesheets here, you'll also need to remove both stylesheets in the _unloadStyleSheets method below.